presto-native-execution/presto_cpp/main/http/HttpServer.cpp

```diff
@@ -148,7 +148,7 @@ HttpServer::HttpServer(
       handlerFactory_(std::make_unique<DispatchingRequestHandlerFactory>()),
       httpExecutor_{std::make_shared<folly::IOThreadPoolExecutor>(
           httpExecThreads,
-          std::make_shared<folly::NamedThreadFactory>("HTTPSrvExec"))} {
+          std::make_shared<folly::NamedThreadFactory>("HTTPSrvIO"))} {
```

Comment:
We could consider to move this executor to Presto server as well so we can have a centralized place to manage all the executors to ease debugging and maintenance. Thanks!

Reply:
lets move it then